lib/msf/core/handler/reverse_tcp_double.rb

```diff
@@ -95,16 +95,22 @@ def start_handler
       sock_inp = nil
       sock_out = nil
 
-      print_status("Started reverse double handler")
+      unless datastore['MODULEOWNER'] == Msf::Exploit::Remote::BrowserAutopwnv2
```

Comment:
Why is ``datastore['MODULEOWNER']`` used? We already have module.owner and various other fields available, including the module instance itself (which can be used as a hash)